sysboundary/antisymmetric.cpp

```diff
@@ -121,6 +125,41 @@ namespace SBC {
             mpiGrid[cells[c]]->sysBoundaryFlag = flag;
          }
       }
+
+         // Assign boundary flags to local fsgrid cells
+      const std::array<int, 3> gridDims(technicalGrid.getLocalSize());  
+      for (int k=0; k<gridDims[2]; k++) {
+         for (int j=0; j<gridDims[1]; j++) {
+            for (int i=0; i<gridDims[0]; i++) {
+               const auto& coords = technicalGrid.getPhysicalCoords(i,j,k);
+               const auto refLvl = mpiGrid.get_refinement_level(mpiGrid.get_existing_cell(coords));
+               if(refLvl == -1) {
+                  cerr << "Error, could not get refinement level of remote DCCRG cell " << __FILE__ << " " << __LINE__ << endl;
+               }
+
+               creal dx = P::dx_ini * pow(2,-refLvl);
+               creal dy = P::dy_ini * pow(2,-refLvl);
+               creal dz = P::dz_ini * pow(2,-refLvl);
+               creal x = coords[0] + 0.5 * P::dx_ini * pow(2,-P::amrMaxSpatialRefLevel);
+               creal y = coords[1] + 0.5 * P::dy_ini * pow(2,-P::amrMaxSpatialRefLevel);
+               creal z = coords[2] + 0.5 * P::dz_ini * pow(2,-P::amrMaxSpatialRefLevel);
+
+               isThisCellOnAFace.fill(false);
+               doAssign = false;
+               
+               determineFace(isThisCellOnAFace.data(), x, y, z, dx, dy, dz);
+               for(uint j=0; j<6; j++) doAssign = doAssign || (facesToProcess[j] && isThisCellOnAFace[j]);
```

Comment:
You are shadowing a int j used in a loop above on line 132. 

Also, why uint, vlasiator normally uses int for these.

Reply:
The uint is copy-paste from previous code (the loop over dccrg cells above this), but I can change it.